scripts/check_cov.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+line_coverage=`lcov --summary coverage/coverage.info | grep "lines......" | awk '/lines/ { print $2 }' | cut -d '%' -f 1`
+line_coverage=$(printf "%.0f" $line_coverage)
+if [ "$line_coverage" -gt 84 ]; then
```

Comment:
line coverage can not set to 90% ?

Reply:
this is no used, codecov bot have method to set threshold, We will enable soon